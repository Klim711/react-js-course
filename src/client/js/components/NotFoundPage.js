import React from 'react';
import { Link } from 'react-router-dom';

import Footer from './Footer';

const NotFoundPage = () => (
  <div className="container">
    <main className="row">
      <h1>Page not found</h1>
      <Link to="/">
          Go to main page
      </Link>
    </main>
    <Footer/>
  </div>
);

export default NotFoundPage;
