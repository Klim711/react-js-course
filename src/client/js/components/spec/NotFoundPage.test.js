import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';

import NotFoundPage from '../NotFoundPage';

describe('NotFoundPage component', () => {
  it('matches to snapshot', () => {
    const renderedValue = renderer.create(
      <BrowserRouter>
        <NotFoundPage/>
      </BrowserRouter>,
    ).toJSON();
    expect(renderedValue).toMatchSnapshot();
  });
});
