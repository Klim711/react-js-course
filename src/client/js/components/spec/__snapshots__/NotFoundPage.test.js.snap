// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component matches to snapshot 1`] = `
<div
  className="container"
>
  <main
    className="row"
  >
    <h1>
      Page not found
    </h1>
    <a
      href="/"
      onClick={[Function]}
    >
      Go to main page
    </a>
  </main>
  <footer
    className="row page-footer"
  >
    <div
      className="title"
    >
      netflixroulette
    </div>
  </footer>
</div>
`;
